readme.md

```diff
@@ -85,6 +85,7 @@
 - [JVM](https://github.com/deephacks/awesome-jvm)
 - [Salesforce](https://github.com/mailtoharshit/awesome-salesforce)
 - [Amazon Web Services](https://github.com/donnemartin/awesome-aws)
+- [Amazon Web Services Alternatives](https://github.com/ro31337/awesome-aws-alternatives)
```

Comment:
You did not add your entry to the bottom of the list as per the [guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md#adding-to-this-list).